src/animation/AnimatedScaleComponent.js

```diff
@@ -22,7 +22,7 @@ export default class AnimatedScaleComponent extends PureComponent<Props, State>
 
   animatedValue = new Animated.Value(this.props.visible ? 1 : 0);
 
-  componentWillReceiveProps(nextProps: Props) {
+  UNSAFE_componentWillReceiveProps(nextProps: Props) {
```

Comment:
The effect of this is basically to acknowledge that we've seen the warning, to tell it to stop warning us.

That's good to do, but before we do we should file an issue about the need to move away from these methods. It can link to that upstream announcement https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html to handle most of the specifics.

Reply:
Makes sense! Opened as #4154.